src/components/card.tsx

```diff
@@ -0,0 +1,21 @@
+import { twJoin } from "tailwind-merge";
+
+type CardProps = {
+  children?: React.ReactNode;
+  className?: string;
+};
+
+export function Card(props: CardProps): React.JSX.Element {
+  const { children, className } = props;
+
+  return (
+    <div
+      className={twJoin(
+        "bg-userCardBG rounded-2xl flex flex-col justify-center items-center shadow-md relative",
+        className
+      )}
```

Comment:
I suggest you to remove the construction of the class inside of className and make it outside of it, assigning it to a variable and passing this variable here.

Reply:
i did that in https://github.com/cristianoprogramador/FE_SH/pull/12

Reply:
Alright, I'll resolve this conversation!